File: iPhone/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/comm/InstTypeTree.m
//
//  InstTypeTree.m
//  VerBank-CSTSv3-IOSClientApi
//
//  Created By Zhanglei on 15/6/29.
//  Copyright (c) 2015年 allone. All rights reserved.
//

#import "InstTypeTree.h"
static NSString * jsonId = @"InstTypeTree";

static NSString * typeName = @"1";
static NSString * parentType = @"2";
static NSString * sort = @"3";
@implementation InstTypeTree
- (id)init{
    self = [super init];
    if(self != nil)
    {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}
@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/CSTS3/comm/struct/InstTypeTree.java
package jedi.ex01.CSTS3.comm.struct;


public class InstTypeTree extends allone.json.AbstractJsonData {

	public static final String jsonId = "10";

	public static final String typeName = "1";
	public static final String parentType = "2";
	public static final String sort = "3";

	public InstTypeTree(){
		super();
		setEntry("jsonId", jsonId);
	}

	public String getTypeName() {
		try {
			String data=getEntryString(InstTypeTree.typeName);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setTypeName(String typeName) {
		setEntry(InstTypeTree.typeName, typeName);
	}

	public String getParentType() {
		try {
			String data=getEntryString(InstTypeTree.parentType);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setParentType(String parentType) {
		setEntry(InstTypeTree.parentType, parentType);
	}

	public int getSort() {
		try {
			int data=getEntryInt(InstTypeTree.sort);
			return data;
		} catch (RuntimeException e) {
			return 0;
		}
	}

	public void setSort(int sort) {
		setEntry(InstTypeTree.sort, sort);
	}


}